test/javascripts/acceptance/chat-transcript-test.js

```diff
@@ -242,6 +264,27 @@ acceptance("Discourse Chat | discourse-chat-transcript", function (needs) {
     );
   });
 
+  test("renders with the reactions attribute", function (assert) {
```

Comment:
I'm not sure if this is considered an acceptance test when it does not test any interaction behaviour of the app.

Reply:
I guess...I really don't want to have to move all this around though lol. But if it's really necessary I can, I guess everything under `acceptance("Discourse Chat | discourse-chat-transcript", function (needs) {` would be an integration test?

Reply:
Hmm maybe something for the future then since this is already the status quo.